web/src/compose_actions.js

```diff
@@ -76,8 +76,7 @@ function clear_box() {
     $("#compose-textarea").removeData("draft-id");
     $("#compose-textarea").toggleClass("invalid", false);
     compose_ui.autosize_textarea($("#compose-textarea"));
-    compose_banner.clear_errors();
-    compose_banner.clear_warnings();
+    compose_banner.clear_all();
```

Comment:
@evykassirer can help with the details, but I don't think this is correct -- we have several notices that are intended to be displayed after sending a message, like the "Sent! " banners in `notifications.js`, and I'm pretty sure this is gonna incorrectly clear those.

Reply:
Should we consider adding a third function to remove the uploads then? This would resolve the issue, and we could use this pull request to discuss changing the separate function calls to a single `clear_all` call later. What do you think?

Reply:
Agreed that we don't want to clear all banners here, unless we also want the message sent banners also being cleared.

Adding a function to clear uploads, or naming a new function that concisely describes the subset of banners we want to close here (if that's something easily named) -- either of these sound fine to me!